src/components/reservation/helper.ts

```diff
@@ -18,9 +18,13 @@ import { Manifestation } from "../../core/utils/types/entities";
 import { PeriodicalEdition } from "../material/periodical/helper";
 import { ModalReservationFormTextType } from "./forms/helper";
 import invalidSwitchCase from "../../core/utils/helpers/invalid-switch-case";
-import { OptionsProps } from "../list-details-dropdown/list-details-dropdown";
 import { SubmitOrderStatus } from "../../core/dbc-gateway/generated/graphql";
 
+export interface OptionsProps {
+  value: string;
+  label: string;
+}
```

Comment:
Consider if this interface is really necessary. 

We may already have something which duplicates this in [our Dropdown component](https://github.com/danskernesdigitalebibliotek/dpl-react/blob/develop/src/components/Dropdown/Dropdown.tsx#L5).